src/test/java/de/tum/in/www1/artemis/PlagiarismCaseIntegrationTest.java

```diff
@@ -51,41 +51,65 @@ public class PlagiarismCaseIntegrationTest extends AbstractSpringIntegrationBamb
 
     private static PlagiarismCase plagiarismCase3;
 
+    private static ArrayList<PlagiarismCase> plagiarismCases;
+
     @BeforeEach
     public void initTestCase() {
-        database.addUsers(3, 1, 1, 1);
+        // We need at least 3 cases
+        plagiarismCases = this.createPlagiarismCases(100);
+        plagiarismCase1 = plagiarismCases.get(0);
+        plagiarismCase2 = plagiarismCases.get(1);
+        plagiarismCase3 = plagiarismCases.get(2);
+    }
+
+    /***
+     * Create a given amount of plagiarism cases
+     * @param numberOfPlagiarismCases The required number of cases
+     * @return The list of generated plagiarism cases
+     */
+    private ArrayList<PlagiarismCase> createPlagiarismCases(int numberOfPlagiarismCases) {
+        var plagiarismCasesList = new ArrayList<PlagiarismCase>();
+        PlagiarismCase plagiarismCase = null;
```

Comment:
We don't need these external declarations since all variables are only used inside the loop. So you can move them directly in the loop.

Reply:
Don't agree, as we can do it more flexible and define the number of the plagiarism cases in the initTestCase method.

Reply:
done